fi.skyjake.Lagrange.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "fi.skyjake.Lagrange",
+    "default-branch": "release",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "lagrange",
+    "rename-desktop-file": "fi.skyjake.lagrange.desktop",
+    "rename-icon": "fi.skyjake.lagrange",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home"
```

Comment:
What does it need home access for?

Reply:
If this is about downloads directory, I'd rather change it to `xdg-download` (and make sure the app follows that environment variable).

Reply:
Yes, the app needs write access to a downloads directory, so `--filesystem=xdg-download` would work for that. Do you suggest checking a `XDG_DOWNLOAD_DIR` environment variable?

The app also needs write access to its XDG config directory. Does that need a separate permission flag or is that allowed by default? Can I rely on `XDG_CONFIG_HOME` being set?

The app needs read-only access to any drag-and-dropped file based on its path. This could be anywhere under the user's home directory, so could `--filesystem=home:ro` be acceptable for that?

Reply:
Yeah, I think following XDG spec is worth regardless of flatpak. `XDG_CONFIG_HOME` (and `XDG_DATA_HOME`) are places your app has write access by default. As for drag&drop, I guess `home:ro` is fine as otherwise it will break.